dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/fixtures/custom_components.js

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+/* eslint-disable strict */
+/**
+ This is a demo custom component used to try out the feature OR to run tests.
+ It is a simple button that will modify the value value to 'test'. Un-minified jsx code below.
+  export default function RegionSelector(props) {
+  const { param, handleBasicFormParamChange } = props;
+
+  const [value, setValue] = useState(props.value || "");
+
+  useEffect(() => {
+    setValue(props.param.value);
+  }, [props.param.value]);
+
+  const handleChange = newValue => {
+    handleBasicFormParamChange(props.param)({
+      currentTarget: {
+        value: "test"
+      }
+    });
+    setValue(newValue);
+  };
+
+  const selectedStyle = {
+    backgroundColor: "rgba(0, 140, 255, 0.19)"
+  };
+
+  return (
+    <>
+      <button
+        key={`test`}
+        type="button"
+        onClick={() => handleChange("test")}
+        style={"test" === value ? selectedStyle : null}
+      >
+        Test
+      </button>
+    </>
+  );
+}
+ */
+!(function (t, e) {
```

Comment:
So a couple of questions:

1. Does the custom component code need to be pre-minified? If not, let's not pre-minify it so users don't need to trust it's not doing something malicious nor go to the effort of verifying it is the minified version of the example code.
2. I understand from the docs that this doesn't need to be under `dashboard/src` at all (where it could at some point easily be bundled in the Kubeapps js by mistake with some unknown minified js doing something bad). I'm strongly against having any chance of pre-minified js code bundled in Kubeapps. Can we instead place this file under `docs/developer/...` next to the documentation (and update the link to it below).

Or let us know why doing so is an issue? Thanks!

Reply:
yeah we can easily move it there! i just put it here since i use it in the tests for the component so i figured it would make more sense here. and it don't not have to be minimized, it's the output from the repo that i mentioned earlier

Reply:
Great, thanks.